.azure/templates/scanners.yml

```diff
@@ -0,0 +1,87 @@
+parameters:
+  BuildFolder: ''
+  SymbolsFolder: ''
+  SourceFolder: ''
+  BuildConfiguration: Release
+  BuildPlatform: x86

```

Comment:
Did you mean for this here?

Reply:
Yes I am expecting consumers of the template to pass or override these arguments.